keep/api/tasks/process_event_task.py

```diff
@@ -302,7 +302,8 @@ def __handle_formatted_events(
         # if new grouped alerts were created, we need to push them to the client
         if grouped_alerts:
             logger.info("Adding group alerts to the workflow manager queue")
-            workflow_manager.insert_events(tenant_id, grouped_alerts)
+            #workflow_manager.insert_events(tenant_id, grouped_alerts)
+            #Inserting of the incident into workflow manager
```

Comment:
Is there a standard way for incidents to be added to the workflow manager or there should be a new method similar to `insert_events`?